gp-nested-forms/gpnf-display-child-entries-table-format-all-child-fields.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Gravity Perks // GP Nested Forms // Display Table Format for All Fields
```

Comment:
Let's update "Display Table Format for All Fields" to "Display Table Format for All Fields and Show All Child Fields".

Reply:
Done